backend/requirements.txt

```diff
@@ -8,4 +8,5 @@ ptvsd==4.3.2
 pymysql==0.10.1
 django-mysql==2.2.0
 djangorestframework==3.12.2
-django-cors-headers==3.3.0
\ No newline at end of file
+django-cors-headers==3.3.0
+requests==2.25.0
```

Comment:
For RequestsClient, used in UTs. I'm running uts with this: `docker-compose run --rm moneywise-backend python manage.py test`, though just `python manage.py test` should work if you have everything installed.